components/NavBar/NavBar.jsx

```diff
@@ -1,52 +1,81 @@
 import React, { useState } from "react";
 import styles from "./NavBar.module.scss";
 import Link from "next/link";
-
+import Alert from "@material-ui/lab/Alert";
```

Comment:
Previously your import was **import Alert from "@material-ui/lab/alert"** with a lower case 'a' , that's why there was a problem with deployment